lib.php

```
@@ -0,0 +1,142 @@
+    'U12' => 'Dzieci',
+);
+
+// ---------------------------------------------------------------------------
+// Divisions: R (Recurve), C (Compound), B (Barebow)
+// ---------------------------------------------------------------------------
+function CreateStandardDivisions($TourId, $TourType) {
+    $i = 1;
+    CreateDivision($TourId, $i++, 'R', 'Łuk klasyczny', 1, 'R', 'R');
+    CreateDivision($TourId, $i++, 'C', 'Łuk bloczkowy',  1, 'C', 'C');
+    CreateDivision($TourId, $i++, 'B', 'Łuk barebow',    1, 'B', 'B');
+}
```

PHPMD (2.15.0)

[error] The parameter $TourId is not named in camelCase. (undefined)

(CamelCaseParameterName)